eng/pipelines/common/platform-matrix.yml

```diff
@@ -229,6 +231,7 @@ jobs:
       helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
       osGroup: Windows_NT
       archType: arm
+      platform: Windows_NT_arm
```

Comment:
This is being fixed by https://github.com/dotnet/runtime/pull/642 also. 

Should we instead close that PR? We might want to run the tests @jashook mentioned there.

Reply:
About why it's here, I imagine we need the artifacts from the platform for this PR. (@jkoritzinsky can confirm.)

Are you referring to running the coreclr outerloop?

I think it would be easiest to drive the other PR to merge because it's more self-contained and probably easier to verify. Then the diff can be removed here.

Reply:
I originally added this just so the job naming for the installer jobs was correct. I didn't realize that this was blocking the ARM testing at that time. We should be able to temporarily remove this and allow #642 to add it.